t/app/controller/contact.t

```diff
@@ -535,6 +535,8 @@ for my $test (
         FixMyStreet::override_config {
             ALLOWED_COBRANDS => [ 'buckinghamshire' ],
         }, sub {
+            my $bucks = $mech->create_body_ok(2217, 'Buckinghamshire Council');
+            ($problem_main) = $mech->create_problems_for_body(1, $bucks->id, 'Test');
```

Comment:
I'd probably stop reusing problem_main if it's just a problem for this subtest.

Reply:
Done in https://github.com/mysociety/fixmystreet/pull/3826/commits/29c1f682b45e06854ac6de2375afe4f343a27b9a